readme.md

```diff
@@ -36,8 +36,8 @@ interface InjectedWindowProvider {
         /** human-readable name of the dApp */
         dAppName: string, 
 
-        /** ID of the key agreement key of the dApp DID to be used to encrypt the session messages */
-        dAppEncryptionKeyId: string, 
+        /** URI of the key agreement key of the dApp DID to be used to encrypt the session messages */
+        dAppEncryptionKeyUri: string, 
```

Comment:
This is a positional parameter, not named parameter, so we can change its name while preserving compatibility, let’s keep it like this.